sbi/inference/base.py

```diff
@@ -298,17 +294,10 @@ def _converged(self, epoch: int, stop_after_epochs: int) -> bool:
 
     def _default_summary_writer(self) -> SummaryWriter:
         """Return summary writer logging to method- and simulator-specific directory."""
-        try:
-            simulator = self._simulator.__name__
-        except AttributeError:
-            simulator = self._simulator.__class__.__name__
 
         method = self.__class__.__name__
         logdir = Path(
-            get_log_root(),
-            simulator,
-            method,
-            datetime.now().isoformat().replace(":", "_"),
+            get_log_root(), method, datetime.now().isoformat().replace(":", "_"),
```

Comment:
the date and time is already encoded in the log dir path by tensorboard. I think it would be good to have a stable simulator name. Then all the different runs for a given simulator are saved in one folder, each under their respective call time.

Reply:
But how? The simulator is no longer passed to the inference object, so we can not infer the name.

Reply:
I mean a name of the `sbi-logs` folder for a given simulator that does not change on every inference call. We are now inferring the simulator name for the filename, no? So this would work then. 
I am just against adding the date and time to the folder name because `tensorboard` will do that anyways. E.g., for a simulator with filename `mysimulator`, and SNPE_C inference it will create a folder structure like this: 
- `/mysimulator/SNPE_C/2020-10-19T08_50_06.298134`
- `/mysimulator/SNPE_C/2020-11-02T10_52_46.466534`
- etc.

Reply:
Maybe I'm misunderstanding, but:

How should we infer the simulator name within `base.py`? The simulator is no longer passed to the inference class, so how should we infer the name of the simulator?

Reply:
OK, now I see. then one alternative solution to adding the date and time would be to move the instantiation of the summary writer to the inference classes. 
But I am fine to leave like this now and to check how well this works in practice.